frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -369,14 +372,42 @@ const SECTIONS = updateSectionsWithPlugins({
         getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
       },
       {
-        widget: EmbeddingCustomizationInfo,
-        getHidden: (_, derivedSettings) =>
-          !derivedSettings["enable-embedding"] ||
-          MetabaseSettings.isEnterprise(),
+        widget: EmbeddingOption,
+        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
+        embedName: t`Standalone embeds`,
+        embedDescription: t`Securely embed individual questions and dashboards within other applications.`,
+        embedType: "standalone",
+      },
+      {
+        widget: EmbeddingOption,
+        getHidden: (_, derivedSettings) => !derivedSettings["enable-embedding"],
+        embedName: t`Full-app embedding`,
+        embedDescription: t`With this Pro/Enterprise feature you can embed the full Metabase app. Enable your users to drill-through to charts, browse collections, and use the graphical query builder.`,
+        embedType: "full-app",
+      },
+    ],
+  },
+  "embedding_in_other_applications/standalone": {
```

Comment:
This is how it'll look like
![image](https://user-images.githubusercontent.com/1937582/176175348-84c9f745-f07f-4604-8c15-81618973af7a.png)